src/StreamrClient.js

```diff
@@ -46,8 +47,12 @@ export default class StreamrClient extends EventEmitter {
             this.options.auth.apiKey = this.options.apiKey
         }
 
-        this.session = new Session(this, options.auth)
-        this.signer = (options.auth && options.auth.publishWithSignature) ? new Signer(options.auth) : undefined
+        if (this.options.privateKey && !this.options.privateKey.startsWith('0x')) {
```

Comment:
Bug? Should be `this.options.auth.privateKey`, right? If this is uncaught by tests, then please add a unit test that gives a `privateKey` without `0x` and asserts that it gets added.